docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -24,52 +26,58 @@ In BODS, the following properties are used to represent such information disclos
 
 * ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
 * ``directOrIndirect`` (See :ref:`schema-interest`)
-* ``componentStatementIDs`` (See :ref:`schema-relationship-record`)
+* ``componentRecords`` (See :ref:`schema-relationship-record`)
 * ``isComponent`` (See :ref:`schema-entity-record`)
 
 Requirements
 ------------------------
 
-The beneficial ownership relationship
+No beneficial ownership relationships
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'noBeneficialOwners'.
```

Comment:
```suggestion
If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty` 'noBeneficialOwners'.
```

I think the structure of this has changed and 'reason' is no longer a named field

Reply:
ahhh my bad I misunderstood that reference